Makefile

```diff
@@ -82,7 +82,7 @@ LIBFILES=stdlib.cma std_exit.cmo *.cmi camlheader
 COMPLIBDIR=$(LIBDIR)/compiler-libs
 
 TOPINCLUDES=$(addprefix -I otherlibs/,$(filter-out %threads,$(OTHERLIBRARIES)))
-RUNTOP=./runtime/ocamlrun$(EXE) ./ocaml$(EXE) \
+RUNTOP = $(OCAMLRUN) ./ocaml$(EXE) \
```

Comment:
I initially thought this was incorrect: if you've just _added_ a primitive and updated the stdlib, then you'd expect to be able to use that primitive. However, that's not correct: while you can add a new primitive to the runtime and use it without _bootstrapping_, you can't do that without repeating _coldstart_.

Therefore, `ocaml$(EXE)` is produced using the `BOOT_OCAMLC` and should therefore be running on `$(OCAMLRUN)` and not `runtime/ocamlrun$(EXE)`.

Aha, but if you're working on a fix in the runtime with no changes to bytecode and primitives, might you expect `make runtop` to use the runtime you're working on, rather than one you booted with? Well, at the moment that doesn't matter because `make runtop` begins with a `coldstart`. However, cf. https://github.com/ocaml/ocaml/pull/10114#pullrequestreview-560715273.

I think the conclusion of my ramble is that above the `$(MAKE) coldstart)` in the `runtop` recipe it might be worth adding a comment that `$(RUNTOP)` could want changing to use `runtime/ocamlrun` if we ever _stop_ doing `coldstart` at the start of `runtop`.